README.md

```diff
@@ -6,32 +6,223 @@
 
 Convert between [itk](https://itk.org) and [napari](https://napari.org) data structures.
 
-Installation
-------------
+## Installation
 
-```
+```sh
 pip install itk-napari-conversion
 ```
 
-Usage
------
+## Usage
 
-Convert an `itk.Image` to an `napari.layers.Image`:
+### Image Conversion
 
-```
+#### Convert ITK Image to napari Image Layer
+
+Convert an `itk.Image` to a `napari.layers.Image`:
+
+```python
 from itk_napari_conversion import image_layer_from_image
 
 image_layer = image_layer_from_image(image)
 ```
 
-Convert to an `napari.layers.Image` to an `itk.Image`:
+**Features:**
+- Automatically detects and handles RGB/RGBA images
+- Preserves image metadata (spacing, origin, direction, custom metadata)
+- Converts ITK's physical space information to napari's layer transformations:
+  - `spacing` → `scale`
+  - `origin` → `translate`
+  - `direction` → `rotate`
+
+**Example:**
+```python
+import itk
+import napari
+from itk_napari_conversion import image_layer_from_image
+
+# Read an image
+image = itk.imread('path/to/image.nrrd')
+
+# Add custom metadata
+image['units'] = 'mm'
+image['patient_id'] = '12345'
+
+# Convert to napari layer
+image_layer = image_layer_from_image(image)
+
+# The layer will have:
+# - image_layer.scale = image spacing (in reverse order for NumPy)
+# - image_layer.translate = image origin
+# - image_layer.rotate = image direction matrix
```

Comment:
`translate` is also in reverse order, but `rotate` is _not_ in reverse order, right? Or are they all in reverse order?

Reply:
I think they are all in reverse order (?) but I have not see documentation on this if it exists. I will add the note to all three.